Ficek_QuantiApp/Ficek_QuantiApp/Domain/Sources/RocketClient/Models/Errors/RocketError.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
```

Comment:
Ten adresar `Errors` bych dal asi na stejnou urovni jako `Models`. Jako je pravda, ze jsou to trochu modely ale taky trochu ne (zalezi na uhlu pohledu) ale tak at zbytecne nevnorujeme...